### DISCUSSION TITLE
TP6 - ¿Como convertir los argumentos como cadena a números?

### DISCUSSION THREAD

**Discussion by damianperezz** [INGCOM-UNRN-P1/cursada-2023#175] [Preguntas y respuestas] [answered]
cuando se pasan argumentos por linea de comando lo que esta entrando al main son cadenas de caracteres, por lo que en el 3 y 4 cuando se piden argumentos que son numeros enteros como se haria para tomarlos en cuenta como numeros y no como cadenas? se podria usar alguna funcion de c?

**Comment by martinvilu** [comment, marked as answer]
https://github.com/orgs/INGCOM-UNRN-P1/discussions/172
